File: libgdx_slam_the_bird/desktop/src/com/nopalsoft/slamthebird/desktop/DesktopLauncher.java
package com.nopalsoft.slamthebird.desktop;

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
import com.nopalsoft.slamthebird.MainSlamBird;

public class DesktopLauncher {
    public static void main(String[] arg) {
        Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
        config.setWindowedMode(480, 800);
        config.setTitle("Slam the bird");
        new Lwjgl3Application(new MainSlamBird(), config);
    }

}

File: Sokoban/desktop/src/com/nopalsoft/sokoban/desktop/DesktopLauncher.kt
package com.nopalsoft.sokoban.desktop

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration
import com.nopalsoft.sokoban.SokobanGame

fun main() {
    val config = Lwjgl3ApplicationConfiguration()
    config.setTitle("Sokoban")
    config.setWindowedMode(800, 480)
    Lwjgl3Application(SokobanGame(), config)
}

